app/routes/home/user/edit.js

```diff
@@ -12,16 +12,20 @@ export default Route.extend(AuthenticatedRouteMixin,{
     return this.store.findRecord('user', params.id);
   },
 
-  setupController(controller){
+  setupController(controller, currentModel){
```

Comment:
Here you can use 
setupController(controller, model){
...
controller.set('firstName', model.get('firstName'));
}

don't need to use this.get('currentModel.firstName'));
￼

Reply:
Done